File: frontDesastres/src/app/shared/model/ong.ts
export class Ong {

    id?: string;
    nome = '';
    email = '';
    telefone = '';
    regiaoAtuacao = '';
    rede_social = '';
    cnpj = '';
    pix = '';


    constructor(id?: string, volunteer: Ong = {nome : '', email: '', telefone: '', regiaoAtuacao: '', rede_social: '', cnpj:'', pix:''}) {
        this.id = id;
        this.nome = volunteer.nome;
        this.email = volunteer.email;
        this.telefone = volunteer.telefone;
        this.regiaoAtuacao = volunteer.regiaoAtuacao;
        this.rede_social = volunteer.rede_social;
        this.cnpj = volunteer.cnpj;
        this.pix = volunteer.pix;
    }
        
}

File: apiDesastres/src/main/java/br/edu/ifpb/gugawag/apiDesastres/model/Ong.java
package br.edu.ifpb.gugawag.apiDesastres.model;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;

@Entity
@Table(name = "t_ong")
public class Ong {

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Long id;

    private String nome;
    private String email;
    private String telefone;
    private String regiaoAtuacao;
    private String rede_social;
    private String cnpj;
    private String pix;

    // Getters e Setters

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getTelefone() {
        return telefone;
    }

    public void setTelefone(String telefone) {
        this.telefone = telefone;
    }

    public String getRegiaoAtuacao() {
        return regiaoAtuacao;
    }

    public void setRegiaoAtuacao(String regiaoAtuacao) {
        this.regiaoAtuacao = regiaoAtuacao;
    }

    public String getRede_social() {
        return rede_social;
    }

    public void setRede_social(String rede_social) {
        this.rede_social = rede_social;
    }

    public String getCnpj() {
        return cnpj;
    }

    public void setCnpj(String cnpj) {
        this.cnpj = cnpj;
    }

    public String getPix() {
        return pix;
    }

    public void setPix(String pix) {
        this.pix = pix;
    }
}
